packages/grafana-data/src/field/fieldDisplay.ts

```diff
@@ -160,7 +160,7 @@ export const getFieldDisplayValues = (options: GetFieldDisplayValuesOptions): Fi
             });
 
             values.push({
-              name,
+              name: '',
```

Comment:
As far as I understand, previously this code incorrectly referenced the (now deprecated) global [name](https://developer.mozilla.org/en-US/docs/Web/API/Window/name) property. Here I've replaced it with an empty string to satisfy the interface, but there might be a more appropriate value.

Reply:
Yep